src/main/webapp/jsp/new-task.jsp

```diff
@@ -11,47 +11,47 @@
 </head>
 <body>
 <div class="newtask">
-    <form>
+    <form action="<%= request.getContextPath() %>/hello" method="post">
```

Comment:
Why you put here pass to `/hello` servlet instead of `/NewTaskServlet`? Since `/NewTaskServlet` post method contains logic to handle it.